.github/workflows/pipeline.yml

```diff
@@ -459,9 +459,9 @@ jobs:
             type=ref,event=pr
             type=sha
             type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}
-            type=raw,value=latest,enable=${{ startsWith(github.ref, 'refs/tags/v3') }}
+            type=semver,pattern={{major}}.{{minor}},enable=${{ !contains(github.ref, '-rc') }}
+            type=semver,pattern={{major}},enable=${{ !contains(github.ref, '-rc') }}
```

Comment:
IMO unnecessary. Those weren't created for some reason for -rc images anyway. I don't know why though 😄

Reply:
done, thank you
https://github.com/langfuse/langfuse/pull/4782